_posts/2021-03-14-openttd-1-11-0-RC1.md

```diff
@@ -0,0 +1,30 @@
+---
+title: OpenTTD 1.11.0-RC1
+author: TrueBrain
+---
+
+Today we release our first Release Candidate of our 1.11 release series!
+
+The last two weeks we added a few more features we wanted to have for our 1.11 series, and fixed many bugs.
+A few entries worth mentioning:
+- Many build windows now have a way to filter lists. This is especially useful if you have many NewGRFs loaded.
+- You can now select the Display Refresh rate and if Hardware Acceleration should be enabled or not from the Game Options.
+- NewGRF Scanner and World Generation now feel a lot smoother and the GUI is more responsive while they are busy.
+- Over 18 bug-fixes; that is more than 1 a day!
+
+With this Release Candidate we will no longer accept new features for 1.11 and we will focus the next two weeks on bug-fixing and making sure 1.11 is as stable as we can get it.
+If you find any bug with this Release Candidate, please report it to us so we can tackle it before the real release.
+
+OpenTTD has been very busy in the last four weeks.
+To put it in words of [GitHub Pulse Insight](https://github.com/OpenTTD/OpenTTD/pulse/monthly):
```

Comment:
Just an observation: The pulse stats are always relative up to the current date, which means everybody following the link will see different stats to the quoted ones. Maybe put in a sentence from which date range the numbers were from?